test/cluster/test_response_timeout_in_send_queue.py

```
@@ -0,0 +1,513 @@
+    # Single shard so the CAS runs locally (no bounce), isolating the write_type.
+    servers = await manager.servers_add(1, cmdline=['--smp=1'])
+    cql, hosts = await manager.get_ready_cql(servers)
+    host_ip = servers[0].ip_addr
+
+    async with new_test_keyspace(manager, "WITH REPLICATION = { 'replication_factor' : '1' }", hosts[0]) as ks:
+        async with new_test_table(manager, ks, "p int PRIMARY KEY, v int", "", hosts[0]) as tbl:
+            async with inject_error(manager.api, host_ip, "transport_expire_response_deadline"):
+                # Conditional INSERT -> LWT/CAS.
+                with pytest.raises(WriteTimeout, match="Request timeout exceeded") as excinfo:
+                    await asyncio.wait_for(
+                        cql.run_async(f"INSERT INTO {tbl} (p, v) VALUES (1, 1) IF NOT EXISTS"), timeout=30.0)
```

Ruff (0.16.2)

[error] Possible SQL injection vector through string-based query construction

(S608)